### PR TITLE
update to Unicode 11.0.0 (bpo-33778)

### PR DESCRIPTION
**Pull request by benjaminp**
https://bugs.python.org/issue33778

### CODE REVIEW

File: Modules/unicodedata.c [hunk 1]
```diff
@@ -921,7 +921,7 @@ is_unified_ideograph(Py_UCS4 code)
 {
     return
         (0x3400 <= code && code <= 0x4DB5)   || /* CJK Ideograph Extension A */
-        (0x4E00 <= code && code <= 0x9FEA)   || /* CJK Ideograph */
+        (0x4E00 <= code && code <= 0x9FEF)   || /* CJK Ideograph */
```
Comment (matrixise) [inline Modules/unicodedata.c:924]: Hi @benjaminp, why did you change the value? just for my own info, I am interested.
Thank you

Comment (ammaraskar) [reply Modules/unicodedata.c:924]: Current version: https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt
Old version: https://www.unicode.org/Public/10.0.0/ucd/PropList.txt

ctrl+f for CJK

Comment (matrixise) [reply Modules/unicodedata.c:924]: thanks for the info.

**Review by matrixise — APPROVED** [review]
